.rubocop.yml

```diff
@@ -1,13 +1,23 @@
+Lint/NestedMethodDefinition:
```

Comment:
@ammazzaw feel free to make some executive decisions about which of these should be enabled/disabled

Reply:
Yep. For now let's stick to the defaults - They should be sensible enough and you've gone through the work of making the linting pass.